src/Reader.php

```diff
@@ -284,6 +284,10 @@ private function getReader($import, $filePath, string $readerType = null, string
 
         $this->currentFile = $this->filePathHelper->getRealPath($filePath, $disk);
 
+        if ($import instanceof ShouldQueue && ($remoteTempDisk = config('excel.imports.remote_temp_disk')) !== null) {
```

Comment:
Make default behaviour, merge with line above. getRealPath should store temp file on a disk + return the tmp file.